cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,67 @@
+import ast
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
+            '--markets',
+            nargs='?',
+            help='String representation of a list of markets'
+        )
+
+    def get_data_snapshots(self):
+        """ Gets all data snapshots from browse pages
+        Assumes there is one data snapshot per page
+        """
+        snapshots = []
+        for page in BrowsePage.objects.all():
+            stream_data = page.specific.content.stream_data
+            snapshot = filter(
+                lambda item: item['type'] == 'data_snapshot',
+                stream_data
+            )
+            if snapshot:
+                snapshot[0]['value']['page'] = page
+                snapshots.append(snapshot[0]['value'])
+        return snapshots
+
+    def find_data_snapshot(self, market_key, snapshots):
+        """ Look up data snapshot by the provided market key
+        Assumes there is one data snapshot per key
+        """
+        snapshot = filter(
+            lambda item: item['market_key'] == market_key,
+            snapshots
+        )
+        if snapshot:
+            return snapshot[0]
+        return None
+
+    def handle(self, *args, **options):
+        # Convert market data represented by a string,
+        # to its python representation (a list of dicts)
+        markets = ast.literal_eval(options['markets'])
+
+        snapshots = self.get_data_snapshots()
+        for market in markets:
+            snapshot = self.find_data_snapshot(market['market_key'], snapshots)
+            if not snapshot:  # Market may not have been added to Wagtail yet
```

Comment:
Failing silently like this might hide problems with the input; consider at least logging something if this happens?